### ISSUE TITLE
Private Session support

### ISSUE THREAD

**Issue by Zero3K** [giannitedesco/project-mayhem#2]
I have done a private session and found that it connects to the following stream URL:

rtmp://fcs6-1.streamate.com/naiad/live34/mp4:p/25221173/show_4319290966_1426352893613.mp4

When in a free session (as a guest), it connects to the following stream URL:

rtmp://fcs97-1.streamate.com/reflect/4319290966/4319290966

When in a free session (as a member), it connects to the following stream URL:

rtmp://fcs6-1.streamate.com/naiad/live34/mp4:f/25221173/show_4319290966.mp4

I hope that information is enough to add support for them.

**Comment by giannitedesco** [comment]
Not quite, those numbers are random. There are several mesage types that need supporting. I have some logs of private sessions if anyone wants to implement that.

Like I said ,right now focussing on paid work I'm afraid!

**Comment by Zero3K** [comment]
But the mp4:f and mp4:p are probably the key to getting it to work, since I'm pretty sure that f equals free and p equals paid (or private). Also, the /mp4:f (or p)/number/ are the same. Its just the number(s) after the show_ that change.

**Comment by giannitedesco** [comment]
Parsing NaiadGoldShow is the key to getting it to work since that's how the client knows the URL's to connect to.

INVOKE(
 'NaiadGoldShow'
 0.000000
 null
 {
    .action = 2.000000
    .duration = 480.000000
    .id = 1131971.000000
    .minbuyin = 300.000000
    .requestedamt = 5000.000000
    .show = 'mp4:p/13193261/show_2112961146_1342764181909.mp4'
    .showtopic = 'Let's play a lil. Don't expect a lady here...I'm your slut now
! Pussy play, ass fingering, sexy striping, doggy style. '
    .timetoend = 395.037000
 }
)

Correct answer from client is:
INVOKE(
 'play'
 0.000000
 null
 'mp4:p/13193261/show_2112961146_1342764181909.mp4'
 -1000.000000
 -1000.000000
)

**Comment by giannitedesco** [comment]
Oh and you need to pledge gold to the show before it starts to even get that message:

INVOKE(
 'NaiadPledgeGold'
 0.000000
 null
 {
    .flags = 0.000000
    .id = 1131971.000000
    .amount = 300.000000
 }
)

Otherwise you just get a notification that the gold show exists without a URL:

INVOKE(
 'NaiadGoldShow'
 0.000000
 null
 {
    .action = 1.000000
    .duration = 480.000000
    .id = 1131971.000000
    .minbuyin = 300.000000
    .requestedamt = 5000.000000
    .show = null
    .showtopic = 'Let's play a lil. Don't expect a lady here...I'm your slut now
! Pussy play, ass fingering, sexy striping, doggy style. '
    .timetoend = 400.878000
 }
)

**Comment by giannitedesco** [comment]
^^ that's actually enough info to implement it
